www-plugins/passff/passff-1.1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="zx2c4 pass manager extension for Firefox (Host Binary)"
+HOMEPAGE="https://github.com/passff/passff"
+SRC_URI="
+		https://github.com/passff/passff/releases/download/${PV}/passff.json -> ${P}.json
+		https://github.com/passff/passff/releases/download/${PV}/passff.py -> ${P}.py
+"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=dev-lang/python-3.4.0:*"
+
+TARGET_DIR="${EROOT}usr/lib/mozilla/native-messaging-hosts"
+
+S="${DISTDIR}"
+
+src_compile() {
+	sed -i "s|PLACEHOLDER|${TARGET_DIR}/${PN}.py|" "${P}.json"
```

Comment:
…you must not modify files in DISTDIR. There's no guarantee that the directory will not be read-only, not to mention you're invalidating the file for next build. So just use `S=${WORKDIR}`, and copy file in `src_unpack()` or use `sed -e ... "${DISTDIR}"/... > ...`.

Also, `|| die` for sed.

Also, `EROOT` is not correct here. `EROOT` is only 'temporary' sub-mount, real path uses `EPREFIX` only.